widgetssdk/src/main/java/com/glia/widgets/di/Dependencies.java

```diff
@@ -196,6 +196,7 @@ public static void init(@NonNull GliaWidgetsConfig gliaWidgetsConfig) {
         controllerFactory.init();
         repositoryFactory.getEngagementRepository().initialize();
         sdkConfigurationManager.fromConfiguration(gliaWidgetsConfig);
+        localeProvider.setCompanyName(gliaWidgetsConfig.companyName);
```

Comment:
It isn't possible to call the `internal` function from the Kotlin class in the Java class.

Reply:
It is interesting that it compiled and worked just fine when I test it locally

Reply:
Also the class itself is also internal but lint is only complaining about the method

Reply:
Fixed